web/dispers/dispers.go

```diff
@@ -25,26 +27,61 @@ func createConcept(c echo.Context) error {
 		return err
 	}
 
-	// Create array of hashes
-	hashes := make([]string, len(in.EncryptedConcepts))
-	for index, element := range in.EncryptedConcepts {
-		out, err := enclave.CreateConcept(element)
+	for i, element := range in.Concepts {
+		err := enclave.CreateConcept(&element)
 		if err != nil {
 			return err
 		}
-		hashes[index] = out
+		in.Concepts[i] = element
 	}
 	return c.JSON(http.StatusOK, dispers.OutputCI{
-		Hashes: hashes,
+		Hashes: in.Concepts,
 	})
 }
 
-func deleteConcept(c echo.Context) error {
-	concept := c.Param("concept")
+func getHash(c echo.Context) error {
 
-	err := enclave.DeleteConcept([]byte(concept))
-	if err != nil {
-		return err
+	strConcepts := strings.Split(c.Param("concepts"), "-")
```

Comment:
There is no indication that we expect concepts with a `-` seperator. Could you add a comment above the function? Also, I would prefer `:`, because you could have concepts like `pique-nique`